ipt_connect/IPTdev/models.py

```diff
@@ -27,31 +27,21 @@ def mean(vec):
 		return 0
 
 def special_mean(vec):
-	if len(vec) in [5, 6]:
-		nreject = 1
-	elif len(vec) in [7, 8]:
-		nreject = 2
-	else:
-		nreject = round(len(vec) / 4.0)
+	nreject = round(len(vec) / 4.0)
```

Comment:
For len(vec) = 6, this returns nreject=2 which is not as the previous behaviour

Reply:
Thank you, you ar right; I should revert this to the explicit one. According to the rules:

> If there are five or six jury members the lowest mark is discarded, if there are seven or eight jury members one lowest and one highest mark is discarded. For larger jury numbers, the number of discarded marks is quarter that of the jury members, rounded up (if this number is odd, one more of the lowest marks is discarded than of the highest).

It gives the following:
```
5 6 7 8 9 10 11 12
1 1 2 2 3  3  3  3
```

Am I right?

Reply:
Yes. Basically the previous implementation did the job correcly, so you can simply revert to it

Reply:
Fixed: https://github.com/nickkolok/ipt_connect/commit/ebe1b1abbeb2d89f5b5bc3c9ddb6c1f7ee967a5f